avocado/utils/service.py

```diff
@@ -775,15 +775,12 @@ class has methods to start/stop services.
     # service_command_generator is explicitly undefined so that we get the
     # NameError on first access
     # pylint: disable=W0601
-    global _service_manager
```

Comment:
Would you please provide more details in the commit message? This is actually pretty serious bugfix so it deserves more description. I'd suggest describing the issue, the steps you did to address and perhaps you should mention it can affect the speed of calling "ServiceManager()" multiple times, but in return it fixes the issue. I agree that caching it doesn't make much sense as the actual call is fairly quick and we don't expect people creating ServiceManagers in loops.